ts/Core/Chart/Chart.ts

```diff
@@ -477,6 +477,27 @@ class Chart {
             this.callback = callback;
             this.isResizing = 0;
 
+            const zooming = optionsChart.zooming;
+
+            if (zooming) {
+                optionsChart.pinchType = pick(
+                    zooming.pinchType,
+                    optionsChart.pinchType
+                );
```

Comment:
Now that we have marked the old options deprecated, I think it would be better if we turn this expression upside down: 
```ts
                zooming.pinchType = pick(
                    zooming.pinchType,
                    optionsChart.pinchType
                );
```
Then we would use the `zooming` option in the implementations, so that the actual implementation is not referring to the deprecated names. Unless that results in something significantly more complex.

Subsequently, we should probably also make the `zooming` property mandatory on the `ChartOptions` interface.

`zooming: Partial<ChartZoomingOptions>;`